lesson2/1-10-squaring.py

```diff
@@ -1,3 +1,3 @@
 if __name__=="__main__":
     for i in range(1,11):
-        print(str(i)+'² =',i**2)
\ No newline at end of file
+        print(str(i)+'² =',i**2)
```

Comment:
Adjust code to PEP8 everywhere.
```suggestion
        print(str(i)+'² =', i**2)
```